test/images/image-util.sh

```diff
@@ -140,24 +140,36 @@ build() {
       ${SED} -i "s|BASEARCH|${arch}|g" $dockerfile_name
     fi
 
-    # copy the qemu-*-static binary to docker image to build the multi architecture image on x86 platform
-    if grep -q "CROSS_BUILD_" Dockerfile; then
-      if [[ "${arch}" == "amd64" ]]; then
-        ${SED} -i "/CROSS_BUILD_/d" Dockerfile
-      else
-        ${SED} -i "s|QEMUARCH|${QEMUARCHS[$arch]}|g" Dockerfile
-        # Register qemu-*-static for all supported processors except the current one
-        echo "Registering qemu-*-static binaries in the kernel"
-        local sudo=""
-        if [[ $(id -u) != 0 ]]; then
-          sudo=sudo
+    # Only the cross-build on x86 is guaranteed by far, other arches like aarch64 doesn't support cross-build
+    # thus, there is no need to tackle a disability feature on those platforms, and also help to prevent from
+    # ending up a wrong image tag on non-amd64 platforms.
+    build_arch=$(uname -m)
+    if [[ ${build_arch} = "x86_64" ]]; then
+        # copy the qemu-*-static binary to docker image to build the multi architecture image on x86 platform
+        if grep -q "CROSS_BUILD_" Dockerfile; then
+          if [[ "${arch}" == "amd64" ]]; then
```

Comment:
Better use `=` instead of `==` since they are equal in bash (but `=` is POSIX). Also same thing with the quotes for the static string. Suggestion:
```shell
if [[ "${arch}" = 'amd64' ]]; then
```

Reply:
Done